clusterloader/run-e2e.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -ex
+
+cd e2e/ && go test -c -o e2e.test
```

Comment:
We shouldn't rely on the fact that the script is being run from the current directory.
Instead we should do something similar as we are doing in:
https://github.com/kubernetes/perf-tests/blob/master/verify/test.sh

which is:
KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
cd ${KUBE_ROOT}